packages/p2p-v2/src/pages/my-profile/screens/MyProfileStats/MyProfileStatsLimitModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect } from 'react';
+import Modal from 'react-modal';
+import { useAdvertiserUpdate } from '@deriv/api';
+import { Button } from '@deriv-com/ui/dist/components/Button';
+import { Text } from '@deriv-com/ui/dist/components/Text';
+import './MyProfileStatsLimitModal.scss';
+
+type TMyProfileStatsLimitModalProps = {
+    currency: string;
+    isModalOpen: boolean;
+    onRequestClose: () => void;
+};
+
+type TCustomStyles = {
+    overlay: ReactModal.Styles['overlay'];
+};
+
+const customStyles: TCustomStyles = {
+    overlay: {
+        alignItems: 'center',
+        backgroundColor: 'rgba(0, 0, 0, 0.72)',
+        display: 'flex',
+        justifyContent: 'center',
+        zIndex: 9999,
+    },
+};
```

Comment:
```suggestion
type TCustomStyles = {
    overlay: ReactModal.Styles['overlay'];
};

const customStyles: TCustomStyles = {
    overlay: {
        alignItems: 'center',
        backgroundColor: 'rgba(0, 0, 0, 0.72)',
        display: 'flex',
        justifyContent: 'center',
        zIndex: 9999,
    },
};
```

@nada-deriv, is it possible to move these to shared folders i.e. `constants`, `types`, or `helpers`, so that we can reuse them else where, because they seem common to all modals

Reply:
lets move it to a helpers file as i'm also using this